packages/aws-cdk-lib/aws-ec2/lib/subnet.ts

```diff
@@ -43,6 +43,13 @@ export abstract class SubnetFilter {
     return new CidrMaskSubnetFilter(mask);
   }
 
+  /**
```

Comment:
NIT: Hey, could you add `@params` for the function argument here?

Reply:
Just added this `@param cidrs List of CIDR ranges to filter subnets from`